git-version.py

```diff
@@ -0,0 +1,11 @@
+from subprocess import check_output
+from datetime import datetime
+
+try:
+    git_version=check_output(['git', 'describe', '--abbrev=8', '--always', '--tags', '--dirty= (modified)']).decode().strip()
```

Comment:
Please can you add also `VERSION` define which has --abbrev=0 and in case of issue it returns `vdev`. Basically the same thing these three lines do.
https://github.com/hardwario/twr-lora-1wire-thermometer/blob/master/platformio.ini#L15
We need to keep the backward compatibility with current `VESRION` define that travels over our proprietary radio.

Reply:
Sure, will do. I may have misunderstood you earlier and assumed VERSION will be set from elsewhere.

Reply:
Done.